index.html

```diff
@@ -146,8 +145,17 @@
     {{ render_tag('framework', game['framework']) }}
 
     {%- if 'info' in game %}<div>{{ game['info'] }}</div>{% endif %}
-    
+
+    {%- set images = [] %}
+    {% for item in game.get('media', []) %}
+      {% if 'image' in item -%}
+        {% do images.append(item['image']) %}
+      {%- endif %}
+    {% endfor -%}
     <div class="gallery" style="display: none"></div>
+    {% if images | length > 0 -%}
+      <script class="gallery-json" type="text/json">{{ images | join(',') }}</script>
```

Comment:
I was thinking about joining together `gallery-raw` and `gallery-json`, but I've looked into it and it's easier the way you did it.